packages/core/cache/index.d.ts

```diff
@@ -1,6 +1,8 @@
 import type {FilePath} from '@parcel/types';
 
 export type {Cache} from './lib/types';
+import type {Cache} from './lib/types';
+
 export const FSCache: {
```

Comment:
Nit (keeping all imports at the top)

```suggestion
import type {FilePath} from '@parcel/types';
import type {Cache} from './lib/types';

export type {Cache} from './lib/types';
export const FSCache: {
```